src/components/Firestore/Requests/RequestDetails/Header.scss

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../index.scss';
+@import './Header_variables.scss';
+
+.Firestore-Request-Details-Header {
+  border-radius: 8px 8px 0 0;
+  width: 100%;
+  background-color: var(--mdc-theme-surface);
+  height: $firestore-request-details-header-height;
+  padding: 0 6px;
+  position: relative;
+  // Shadow border at bottom
+  box-shadow: 0 -1px 0 rgba(0, 0, 0, 0.12) inset;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.Firestore-Request-Details-Header-Return {
+  padding-right: $firestore-request-details-header-return-padding-right;
```

Comment:
inline these and use shorter variable names please (since they are not meant to be exported for use in other components)